lib/linter.js

```diff
@@ -308,6 +308,9 @@ class Linter {
     let hasBOM = options.source.charCodeAt(0) === 0xfeff;
     let source = hasBOM ? options.source.slice(1) : options.source;
 
+    // Strips doctype from the source if any
```

Comment:
I'm not sure this comment adds any value, as it's clear from the code below what it's doing. Can we remove it?